### ISSUE TITLE
Pull request processing is failing and unable to post result back to github

### ISSUE THREAD

**Issue by harshad16** [AICoE/sefkhet-abwy#104]
**Describe the bug**
While handling the pull_request webhook event, the post API to respond back to github is having trouble. 
https://github.com/AICoE/sefkhet-abwy/blob/f7f439a0e7ccd52d17c9656271c832d708b0f11e/aicoe/sesheta/review_manager.py#L183

**Additional context**
```
An unhandled exception happened while running webhook event handlers for \"pull_request\"...
Traceback (most recent call last):
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/routing/webhooks_dispatcher.py\", line 84, in route_github_event
    return await github_app.dispatch_event(github_event)
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/github/api/app_client.py\", line 64, in dispatch_event
    return await github_event.dispatch_via(
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/github/models/events.py\", line 146, in dispatch_via
    await aio_gather(*(
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/utils/asynctools.py\", line 54, in aio_gather
    result_pairs_gen = [_r async for _r in _aio_gather_iter_pairs(*aio_tasks)]
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/utils/asynctools.py\", line 54, in <listcomp>
    result_pairs_gen = [_r async for _r in _aio_gather_iter_pairs(*aio_tasks)]
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/utils/asynctools.py\", line 43, in _aio_gather_iter_pairs
    yield await task_res_q.get()
  File \"/opt/app-root/lib64/python3.8/site-packages/anyio/_backends/_asyncio.py\", line 392, in __aexit__
    raise exceptions[0]
  File \"/opt/app-root/lib64/python3.8/site-packages/anyio/_backends/_asyncio.py\", line 415, in _run_wrapped_task
    await func(*args)
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/utils/asynctools.py\", line 21, in _send_task_res_to_q
    task_res = await aio_task
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/routing/routers.py\", line 79, in dispatch
    await aio_gather(*callback_coros)
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/utils/asynctools.py\", line 54, in aio_gather
    result_pairs_gen = [_r async for _r in _aio_gather_iter_pairs(*aio_tasks)]
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/utils/asynctools.py\", line 54, in <listcomp>
    result_pairs_gen = [_r async for _r in _aio_gather_iter_pairs(*aio_tasks)]
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/utils/asynctools.py\", line 43, in _aio_gather_iter_pairs
    yield await task_res_q.get()
  File \"/opt/app-root/lib64/python3.8/site-packages/anyio/_backends/_asyncio.py\", line 392, in __aexit__
    raise exceptions[0]
  File \"/opt/app-root/lib64/python3.8/site-packages/anyio/_backends/_asyncio.py\", line 415, in _run_wrapped_task
    await func(*args)
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/utils/asynctools.py\", line 21, in _send_task_res_to_q
    task_res = await aio_task
  File \"review-manager\", line 183, in on_pr_open_or_edit
    await github_api.post(
  File \"/opt/app-root/lib64/python3.8/site-packages/octomachinery/github/api/utils.py\", line 45, in async_function_wrapper
    return await coroutine_instance
  File \"/opt/app-root/lib64/python3.8/site-packages/gidgethub/abc.py\", line 175, in post
    data, _ = await self._make_request(
TypeError: _make_request() got an unexpected keyword argument 'content_type'
```

**Comment by harshad16** [comment]
octomachinery utilizes the gidgethub , with the gidgethub v4.2.0  the GithubAPI call making function `make_request` has an additional parameter, `context_type` added to it. 
https://github.com/brettcannon/gidgethub/blob/27c104de7a3e1b3052fb252f11b61a8ee01691f1/gidgethub/abc.py#L64
However is this not handled by the octomachinery function call , resulting in a Type Error, due to missing argument 
https://github.com/sanitizers/octomachinery/blob/66ced1dcf50ff8c78021e1bcc01c48d56dcfc865/octomachinery/github/api/raw_client.py#L66

**Comment by harshad16** [comment]
Created an upstream issue, to handle this issue properly. 
https://github.com/sanitizers/octomachinery/issues/45

**Comment by harshad16** [comment]
fixed with octomaching v0.3.5
